mentors/index.md

```diff
@@ -5,18 +5,11 @@ title: Mentors
 
 # Mentors
 
-## Wojciech (Mario) Trocki
+## Laura Fitzgerald
 
-![Picture](https://avatars3.githubusercontent.com/u/981838?v=3&s=160)
 
-[twitter/pantroken](https://twitter.com/pantroken)
+## Niall Kane
 
-## Luigi Zuccarelli
-![Picture](https://avatars0.githubusercontent.com/u/12412882?v=3&s=160)
 
-[github/luigizuccarelli](https://github.com/luigizuccarelli)
+## Matt Sreedaran
```

Comment:
Mathana Sreedaran (Matt)
^ A rare occasion where my actual first name is used :D